src/ui/components/SecondaryToolbox/NewConsole.tsx

```diff
@@ -139,29 +139,9 @@ function FocusContextReduxAdapter({ children }: PropsWithChildren) {
   const [deferredFocusRegion, setDeferredFocusRegion] = useState<FocusRegion | null>(null);
 
   useEffect(() => {
-    if (focusRegion === null) {
-      return;
-    }
-
-    const updateFocusRegionOnceLoaded = () => {
-      const focusBegin = displayedBeginForFocusRegion(focusRegion);
-      const focusEnd = displayedEndForFocusRegion(focusRegion);
-      const isLoaded = loadedRegions?.loaded?.some(region => {
-        return region.begin.time <= focusBegin && region.end.time >= focusEnd;
-      });
-      if (isLoaded) {
-        startTransition(() => {
-          setDeferredFocusRegion(focusRegion);
-        });
-      } else {
-        timeoutId = setTimeout(updateFocusRegionOnceLoaded, 250);
-      }
-    };
-
-    let timeoutId = setTimeout(updateFocusRegionOnceLoaded, 250);
-    return () => {
-      clearTimeout(timeoutId);
-    };
```

Comment:
This should not be necessary. We should be able to load _some_ of our UI without being in a loaded region. Only certain API requests require this.

Also this check wasn't 100% reliable before – for the same reason mentioned above.

Reply:
Josh says this is the not the right values to be reading. There is another selector that returns both time and execution point.

Reply:
Can I put points in the Focus context instead (and update the new app to do what `Timeline` does in terms of eagerly reading these)?

Reply:
Related commit: https://github.com/replayio/devtools/commit/aa62c1e8a1aac5cef19453014894023735f05df2